Fixes #66: GitTips missing visual (#71)
comp129/comp55: docs/labs/GitTips.md

    (You can just use the option "push upstream".
    If it won't let you, continue the pull and merge.

    Here is what a conflicting merge looks like:

    ![Egit-0 10-merge-conflict](https://github.com/comp129/comp55/assets/76021136/2ffb97bc-4b48-4e98-a12d-9afdac298a02)

    You will have to coordinate with your team which lines to keep and which lines to delete.

>     You will also have to remove the  "<<<<< HEAD", "======", and ">>>>>>".

Reviewer: ```suggestion
    You will also have to remove the  `<<<<< HEAD`, `======`, and `>>>>>>`.
```

>     These markers delineate the conflicting changes.

Reviewer: maybe provide a different version or try to link to the Team Git lab, as we have some information there that helps people understand what is actually between those different propoerties.
I think I would want you to mention something about what it means, as many folks my interpret that they need to go through both sets of changes, when the majority of times you'll want one set of changes or the other.
You can use your image as an example here, you could also link potentially to the other lab as well (see how this is done in other labs to do this correctly)

>    Make sure you're selecting one set of changes instead of combining both changes.

Reviewer: ```suggestion
   Typically, you'll select one set of changes but on occasion you may need to select changes from both.
```

>    To get a better understanding of this, you can go to step 8 in [here](https://github.com/comp129/comp55/blob/main/docs/labs/9-Github.md).

Reviewer: ```suggestion
   To get a better understanding of this, you can go to [step 8 of the GitHub Lab](9-Github.html#step-8-martyr--hackslash---commit-and-push-the-same-line).
```
